scripts/kind-e2e/e2e.sh

```diff
@@ -5,6 +5,21 @@ source $(git rev-parse --show-toplevel)/scripts/lib/debug_functions
 
 ### Functions ###
 
+function kind_fixup_config() {
```

Comment:
Maybe we can make these changes in submariner-io/submariner e2e.sh, then consume the fixes via https://github.com/submariner-io/submariner-operator/pull/248? I think the goal is to get rid of these scripts in submariner-operator ASAP. We could also try to get in https://github.com/submariner-io/submariner/pull/425 in first to take advantage of that refactoring.

Reply:
Or second-thinking, maybe it's better just to fix it here for now and do this and other migrations when we move to shipyard.